### DISCUSSION TITLE
Pretty extensive application built with pygubu-designer

### DISCUSSION THREAD

**Discussion by AJ6CU** [alejandroautalan/pygubu-designer#199] [Show and tell]
Hi, 

Just released a pretty extensive application built with pygubu-designer. Could not have done this without this wonderful tool! 

The Settings Editor is a tool to customize the EEPROM settings of the uBITX Amateur Radio kit (hfsignals.com) that are running the KD8CEC software (v1.2 is at https://github.com/phdlee/ubitx) and v2 work continuing at: https://github.com/AJ6CU/ubitxV20

The source code is all here (including the .ui files).
https://github.com/AJ6CU/uBITX-EEPROM-Manager/tree/main/uBITX_Settings_Editor

Installation packages has been made for MacOS, Windows, and Linux (using pyinstaller). Except for some minor tweaks (fonts etc.) all the work in pygubu-designer ported from platform to platform with no changes.

Although the draft manual is here:
https://github.com/AJ6CU/uBITX-EEPROM-Manager/blob/main/uBITX_Settings_Editor/uBITX%20Settings%20Editor%20User%20Manual.pdf

Let me share a few screen shots below to demonstrate what you can build! (ignore the numbers in circles, they are referenced in the user manual)

Regards

Mark
AJ6CU

![afterloading](https://user-images.githubusercontent.com/70183884/226211053-1d9b3d37-5617-4b9d-9448-f7b2a1fafbec.JPG)

![smeterwizard](https://user-images.githubusercontent.com/70183884/226211105-194d11eb-596d-4a1a-a223-025afbeb30fb.jpg)
![displaytab](https://user-images.githubusercontent.com/70183884/226211129-9d837ccf-21e3-4d00-9c7b-9bce045e90d4.jpg)

**Comment by alejandroautalan** [comment]
Hello @AJ6CU, I just saw your post.

I am very glad that pygubu is useful for you. This drives me to keep improving pygubu and pygubu-designer.
Thank you for sharing it!

Later, I'll explore your repository and take a closer look at the ui files.

Regards
Alejandro A.

**Comment by AJ6CU** [comment]
Alejandro,

 

Let me know if you have any suggestions!  Obviously a first timer and probably did a lot of things the hard way. The ironic thing is that I have been involved with marketing GUI builders for most of my life, but this is the first time I actually did something with one!

 

Regards,

 

Mark
